### PR TITLE
Bookstore: create and remove books

### PR DESCRIPTION
**Pull request by kingqabil**
- [x] Installed the necessary dependencies
- [x] Configured the redux store
- [x] Created action types and creators
- [x] Included the Provider component and initialized it to the store
- [x] Created a submitBook handler function and dispatched to store
- [x] Created the Book and BooksList components
- [x] Subscribed to the store in the BookList component using the useSelector

### CODE REVIEW

File: src/components/Book.js [hunk 1]
```diff
@@ -0,0 +1,72 @@
...
+          {chapter}
+        </span>
+        <Button text="UPDATE PROGRESS" handleClick={handleClick} />
+      </div>
+    </div>
+  );
+};
+
+Book.propTypes = {
+  id: PropTypes.string.isRequired,
```
Comment (rloterh) [inline src/components/Book.js:65]: - [ ] Book details are not been added/ stored correctly. This is likely due to proptype associated with book object. Kindly ensure to fix this issue

**Error:**
image

**Review by rloterh — APPROVED** [review]
Hi Sadiq, 

You did extremely great on this project! 👏 👏 
There are however a few issues that you still need to work on to go to the next project but you are almost there! 🚦  🚦  

## Good Points 👍  
- Descriptive Pull Request
- No linter errors


## Required Changes ♻️
_Kindly check the comments under the review._

<br>
Cheers and Happy coding!👏 💻 🍷

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have 2 more limited reviews per this project. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/components/Book.js (hunk 1 above)
Comment (Emmyn5600) [reply src/components/Book.js:65]: Solved

db4

**Review by Emmyn5600 — APPROVED** [review]
Hi @kingqabil  

Good Job on implementing the add authorization rules milestone.

Woo ow 👏, Your project is complete!  There is nothing else to say other than... it's time to merge it. 👏

Congratulations! 🎉 

##  Optional suggestion

- N/A

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**Pull request merged**